Jenkinsfile

```diff
@@ -1,30 +1,47 @@
 pipeline {
     agent any 
     stages {
-	stage('Check Docker image') {
-	    steps {
-		echo "Checking if the docker image is available (logic not implemented)"
-		sh """
-		docker images sungeunbae/qcore-ubuntu-minimal -q
-		"""
-	    }
-	}
+        stage('Settin up env') {
+            steps {
+                echo "Start virtual environment"   
+                sh """ 
+                    pwd
+                    env
```

Comment:
If we're using these as debug output for the logs, it would be nice to wrap it with "Current working directory:" etc.

Reply:
You mean as a separate "stage"? Sure - can do.

Reply:
Done in the latest patch